cmake/FindPulseAudio.cmake

```diff
@@ -0,0 +1,55 @@
+include(FeatureSummary)
+set_package_properties(PulseAudio PROPERTIES
+    URL "https://www.freedesktop.org/wiki/Software/PulseAudio/"
+    DESCRIPTION "sound server system for POSIX OSes"
+)
+
+set(PulseAudio_PKG_CONFIG_SPEC "libpulse>=5.0")
```

Comment:
This one seems broken: I have libpulse 0.9.21 and I got:
```
-- Found PulseAudio: /usr/lib/libpulse.so  
-- dynamic libpulse -> libpulse.so.0
```
And the build failed, naturally.

Reply:
I reverted the use of custom CMake modules in favor of only using pkg_check_modules.
The CMake modules did not test version ranges.

Reply:
That seem to have fixed it.